tests/pytorch_tests/model_tests/feature_models/linear_function_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import torch
+import torch.nn.functional as F
+from tests.pytorch_tests.model_tests.base_pytorch_test import BasePytorchTest
+from model_compression_toolkit.core.pytorch.pytorch_device_config import get_working_device
+
+"""
+This test checks the linear functional substitution function.
+"""
+
+
+class LinearFNet(torch.nn.Module):
+    def __init__(self):
+        super(LinearFNet, self).__init__()
+        self.fc1 = torch.nn.Linear(in_features=1000, out_features=100, bias=False)
+        self.fc2 = torch.nn.Linear(in_features=100, out_features=10, bias=True)
+
+    def forward(self, x):
+        x = F.linear(x, self.fc1.weight, self.fc1.bias)
+        y = F.linear(x, self.fc2.weight, self.fc2.bias)
```

Comment:
change to:
`y = F.linear(x, bias=self.fc2.bias, weight=self.fc2.weight)`
to make sure everything still works